examples/python/qft.py

```diff
@@ -63,9 +63,9 @@ def qft(circ, q, n):
 for j in range(5):
     qft5.measure(q[j], c[j])
 
-print(qft3.qasm())
-print(qft4.qasm())
-print(qft5.qasm())
+print(qft3.draw())
+print(qft4.draw())
+print(qft5.draw())
```

Comment:
Since `__str__` has been overloaded, just `print(qft5)` should work. But this is fine too.